packages/dataprovider/README.md

```diff
@@ -196,11 +191,85 @@ buildGraphQLProvider({
     },
   },
 })
-
+```
+You can also have separate fragments for "one" record and for "many" records (e.g. different views for detail of a resource and for their list):
+```ts
+// real world example
+buildGraphQLProvider({
+  clientOptions: { uri: "/api/graphql" } as any,
+  resourceViews: {
+    ParticipantsToInvoice: {
+      resource: "ChallengeParticipation",
+      fragment: {
+        one: gql`
+          fragment OneBilling on ChallengeParticipation {
+            challenge {
+              title
+            }
+            user {
+              email
+              firstname
+              lastname
+              school {
+                name
+                address
+                city {
+                  name
+                  zipCode
+                  canton {
+                    id
+                  }
+                }
+              }
+            }
+            teamsCount
+            teams {
+              name
+            }
+          }
+        `,
+        many: gql`
+          fragment ManyBillings on ChallengeParticipation {
+            challenge {
+              title
+            }
+            user {
+              email
+              firstname
+              lastname
+              school {
+                name
+                address
+              }
+            }
+            teams {
+              name
+            }
+          }
+        `,
+      }
+    },
+  },
+})
+```
+Now you have a new virtual resource `ParticipantsToInvoice` that can be used to display a List or for one record. (notice: update/create/delete is currently not specified, so use it read-only) and it will have exactly this data.
+
+There are two ways you can use this new virtual resource. If you want to use it with React-Admin's query hooks (`useQuery, useGetList, useGetOne`), you need to add this as a new `<Resource>`, because these hooks rely on Redux store and it will throw an error if the resource is unknown:
+```jsx
+<Admin>
+  // ...
+  <Resource name="ParticipantsToInvoice" />
+</Admin>
```

Comment:
the sentence above suggests that `<Resource name="ParticipantsToInvoice" />` will throw an error because its unknown. But actually `<Resource name="ParticipantsToInvoice" />` is the solution, right?

its maybe a bit missleading (or i understand it incorrectly)

Reply:
Right, that does make sense. I'll try to reword, just a minute.